client/my-sites/plans/jetpack-plans/utils.ts

```diff
@@ -134,23 +134,45 @@ export function getProductWithOptionDisplayName(
 	return slugToSelectorProduct( optionSlug )?.displayName || item.displayName;
 }
 
-// Takes any annual Jetpack product or plan slug and returns its corresponding monthly equivalent
-function getMonthlySlugFromYearly( yearlySlug: string | null ) {
+function getSlugInTerm( yearlySlug: string | null, slugTerm: Duration ) {
```

Comment:
Small nitpick here, but given the way we're calling this function further down, aren't we always returning the slug that matches the  _opposite_ term?

If this is the case, I wonder if it would make more sense to reverse the logic so that when we say, e.g., `getSlugInTerm( 'jetpack_scan', TERM_MONTHLY )`, it returns the monthly slug for Jetpack Scan.

Reply:
You're absolutely right, Matt. I'm going to reverse the logic and push a new commit.

Reply:
Done.